spec.bs

```diff
@@ -2664,6 +2691,81 @@ To <dfn>fetch trusted signals</dfn> given a [=URL=] |url|, an [=origin=] |script
   1. Return « |signals|, |perInterestGroupData|, |dataVersion| ».
 </div>
 
+<div algorithm>
+To <dfn>fetch trusted key value signals</dfn> given a [=URL=] |url|, a [=byte sequence=] |body|, a [=string=]
+|context|, an [=origin=] |scriptOrigin|, a [=policy container=] |policyContainer|, an |indexMap|, and a
+[=boolean=] |isBiddingSignal|:
+
+1. Let |request| be a new [=request=] with the following properties:
+  :   [=request/URL=]
+  ::  |url|
+  :   [=request/method=]
+  ::  `POST`
+  :   [=request/body=]
+  ::  |body|
+  :   [=request/origin=]
+  ::  |scriptOrigin|
+  :   [=request/header list=]
+  ::  «`Accept`: `application/json`»

```

Comment:
It does not seem to be json, but ["message/ad-auction-trusted-signals-response"](https://crsrc.org/c/content/browser/interest_group/trusted_signals_fetcher.cc;drc=5103ac6102b144aff2bb3b45fe525521ecf3320d;l=397).
Seems we also have ["content-type"](https://source.chromium.org/chromium/chromium/src/+/main:content/browser/interest_group/trusted_signals_fetcher.cc;l=407;drc=5103ac6102b144aff2bb3b45fe525521ecf3320d;bpv=1;bpt=1) headers (`message/ad-auction-trusted-signals-request`)?

Reply:
Good catch!

Reply:
Good catch